src/core/plugins/system.ts

```diff
@@ -767,7 +732,6 @@ class System extends CallablePlugin {
     }
   }
 
-  @remoteMethod(RemoteMethods.Undeploy)
   async __undeployApp(data: {appId, deploymentTimestamp}, callerInfo) {
     if(!callerInfo.isDeployer)
```

Comment:
We might need to call this function on non-deployers sometimes. This condition is unnecessary.